packages/scoutgame-ui/src/components/common/Header/HeaderMessage.tsx

```diff
@@ -10,14 +16,25 @@ export function HeaderMessage() {
   const [timeLeftStr, setTimeStr] = useState(getTimeLeftStr());
   const isMounted = useIsMounted();
   const draftSeason = isDraftSeason();
+  const now = DateTime.now();
+  const isSaturday = now.weekday >= 6;
```

Comment:
Should rename to "isSaturdayOrSunday" or "draftWeekEnded".. i was confused by "isSaturday" because i thought you needed to handle Sunday too :)